package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "build": "node ./scripts/verify-packages-versions.js && run-s build:transpile build:types build:bundle",
+    "build": "node ./scripts/verify-packages-versions.js && lerna run build:transpile,build:types,build:bundle",
```

Comment:
Can we add `verify-packages-versions.js` as a dependency to one of the build tasks in `nx`?

Reply:
Yeah this verify-packages-versions is a pretty big eyesore for me. IMO it should just be its own CI job. I might do this in a follow-up pr.

Reply:
Sounds good to me!